Description: default to not monitoring so that regular LVM operation does not
 require dmeventd running.
Author: Kees Cook <kees@ubuntu.com>

Index: merged/lib/config/defaults.h
===================================================================
--- merged.orig/lib/config/defaults.h	2010-02-11 15:56:50.551735761 -0800
+++ merged/lib/config/defaults.h	2010-02-11 15:57:12.971034511 -0800
@@ -48,7 +48,7 @@
 #define DEFAULT_MIRROR_LOG_FAULT_POLICY "allocate"
 #define DEFAULT_MIRROR_DEV_FAULT_POLICY "remove"
 #define DEFAULT_DMEVENTD_MIRROR_LIB "libdevmapper-event-lvm2mirror.so"
-#define DEFAULT_DMEVENTD_MONITOR 1
+#define DEFAULT_DMEVENTD_MONITOR -1
 
 #define DEFAULT_UMASK 0077
 
